urls.py

```diff
@@ -111,6 +111,16 @@
     'distro/core/qiime201711-1512067591.zip':
         'https://s3-us-west-2.amazonaws.com/qiime2-data/distro/core/qiime201711-1512067591.zip',
 
+    # 2017.12 DISTRO
+    'distro/core/qiime2-2017.12-conda-osx-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.12/release/qiime2-2017.12-conda-osx-64.txt',
+    'distro/core/qiime2-2017.12-conda-linux-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.12/release/qiime2-2017.12-conda-linux-64.txt',
+    'distro/core/2017.12':
+        'https://google.com',
+    'distro/core/qiime201711-1512067591.zip':
```

Comment:
This item will overwrite the 2017.11 virtualbox link (line 111 above). I'll update this to something like `distro/core/qiime201712-BUILDID.zip` (matches what we did in previous releases).